assets/js/qa.js

```diff
@@ -0,0 +1,47 @@
+var currentQuestion = 0;
+var score = 0;
+var totQuestions = questions.length;
+
+
+var container =document.getElementById("quizContainer");
+var questionEl = document.getElementById("question");
+var opt1 = document.getElementById("opt1");
+var opt2 = document.getElementById("opt2");
+var opt3 = document.getElementById("opt3");
+var opt4 = document.getElementById("opt4");
+var nextButton = document.getElementById("nextButton");
+var resultCont = document.getElementById("result");
+
+
+function loadQuestion (questionIndex) {
+    var q = questions[questionIndex];
+    questionEl.textContent = (questionIndex + 1) + '.' + q.question;
```

Comment:
Perhaps add a space after the dot?

```suggestion
    questionEl.textContent = (questionIndex + 1) + '. ' + q.question;
```